include/swift/AST/Types.h

```diff
@@ -1353,6 +1353,11 @@ class TupleTypeElt {
   TupleTypeElt getWithType(Type T) const {
     return TupleTypeElt(T, getName(), isVararg());
   }
+
+  /// Retrieve a copy of this tuple type element with the name replaced.
+  TupleTypeElt getWithName(Identifier name = Identifier()) const {
```

Comment:
Nitpick: It seems weird to be able to call `getWithName()` and not have to explicitly specify "no name".